Meadowlark-js/packages/meadowlark-core/src/handler/DescriptorLoader.ts

```diff
@@ -116,23 +111,15 @@ async function loadParsedDescriptors(descriptorData: XmlDescriptorData): Promise
         continue;
       }
 
-      const descriptorDocument: DescriptorDocument = decapitalizeKeys(descriptorReference) as DescriptorDocument;
       const resourceInfo: ResourceInfo = {
         resourceName: descriptorName,
         projectName: 'Ed-Fi',
         resourceVersion: '3.3.1-b',
         isDescriptor: true,
       };
 
-      const documentInfo: DocumentInfo = {
-        ...newDocumentInfo(),
-        documentIdentity: [
-          {
-            name: 'descriptor',
-            value: `${descriptorDocument.namespace}#${descriptorDocument.codeValue}`,
-          },
-        ],
-      };
+      const descriptorDocument: DescriptorDocument = decapitalizeKeys(descriptorReference) as DescriptorDocument;
```

Comment:
DescriptorLoader is the only place the descriptor document shape and identity was defined. Pulled out a few pieces to share.